packages/runtime/container-runtime/garbageCollection.md

```diff
@@ -1,28 +1,38 @@
 # Garbage Collection
-Garbage collection (GC) identifies Fluid objects that are not used and deletes them from the Fluid document. This reduces the size of the Fluid file, the in-memory content and the summary that is uploaded to / downloaded from the server. It also makes processing faster as there is less data to process.
+Garbage collection (GC) is the process by which Fluid Framework performs automatic memory management. GC will run periodically and safely delete objects that are not used. The only responsibility of the users of Fluid Framework is to add and remove references to Fluid objects correctly.
 
-Before understanding the details of how GC works, lets take a look at how to add a reference to Fluid objects when they are in use and remove the reference when they are not in use.
+## Why have Garbage Collection?
+GC reduces the size of the Fluid file, the in-memory content and the summary that is uploaded to / downloaded from the server. It saves COGS on the server and it makes containers load time faster as there is less data to download and process.
 
-## Fluid object references
-- All Fluid objects that are in use must be marked as referenced so that they are not deleted by GC. There are 2 ways to mark objects as referenced:
-  - Create them as `root`. These objects are always referenced and cannot be marked unreferenced later. For example, `root` data stores are always referenced.
+## What do I need to do?
+All Fluid objects that are in use must be marked as referenced so that they are not deleted by GC. Similarly, references to all unused Fluid objects should be removed so that they can be deleted by GC. It is the responsibility of the users of Fluid Framework to correctly add and remove references to Fluid objects.
```

Comment:
```suggestion
All Fluid objects that are in use must be properly referenced so that they are not deleted by GC. Similarly, references to all unused Fluid objects should be removed so that they can be deleted by GC. It is the responsibility of the users of Fluid Framework to correctly add and remove references to Fluid objects.
```